ckanext/ontario_theme/templates/internal/home/snippets/search.html

```diff
@@ -10,8 +10,8 @@
   <form class="module-content search-form" method="get" action="{% url_for controller='package', action='search' %}">
     <div class="search-input form-group search-giant">
       <input aria-label="{% block header_site_search_label %}{{ _('Search datasets') }}{% endblock %}" id="field-main-search" type="text" class="form-control" name="q" value="" autocomplete="off" placeholder="{% block search_placeholder %}{{ placeholder }}{% endblock %}" />
-      <button type="submit">
-        <i class="fa fa-search"></i>
+      <button type="submit" aria-label="{{ _('Search') }}">
```

Comment:
This is a bit duplicative but manual testing with assistive technology would help see a bit more. adding the aria-label while using an inner element as the descriptive text may cause "search" to be read / identified twice.